packages/main/src/plugin/container-registry.spec.ts

```diff
@@ -2147,14 +2147,14 @@ describe('createAndStartContainer', () => {
       api: fakeDockerode,
     } as InternalContainerProvider);
 
-    const container = await containerRegistry.createAndStartContainer('podman1', {});
+    const container = await containerRegistry.createContainer('podman1', { startOnCreation: true });
```

Comment:
should it be 
```suggestion
    const container = await containerRegistry.createContainer('podman1', { start: true });
```

Reply:
I thought you added all changes. I just committed them and went to have lunch. Checking everything now

Reply:
yes it's my fault I didn't suggested changes everywhere